linode/lke/cluster.go

```diff
@@ -337,6 +338,10 @@ func matchPoolsWithSchema(pools []linodego.LKENodePool, declaredPools []interfac
 				continue
 			}
 
+			if !slices.Equal(helper.ExpandStringSet(declaredPool["tags"].(*schema.Set)), apiPool.Tags) {
```

Comment:
Since the tags are unordered, would it make sense to sort both tag slices before comparing here?

Reply:
@lgarber-akamai good point! I checked the underlying docs and Terraform SDK already handle sorting in their set's `List()` method.

> The order of the returned elements is deterministic. Given the same set, the order of this will always be the same. 

https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2@v2.34.0/helper/schema#Set.List

Reply:
@lgarber-akamai good point! I thought TF SDK handled it but I forgot another side is from the API..